vendor/assets/stylesheets/susy/susy/language/susy/_background.scss

```diff
@@ -0,0 +1,386 @@
+    $this-stop: ();
+
+    $start: parse-span(1 at $location $grid);
+    $start: get-isolation($start);
+    $this-stop: append($this-stop, transparent $start, comma);
+    $this-stop: append($this-stop, $_color $start, comma);
+
+    $_end: $start + span(1 at $location $grid);
+    $this-stop: append($this-stop, $_light $_end, comma);
+    $this-stop: append($this-stop, transparent $_end, comma);
+
+    $_stops: join($_stops, $this-stop, comma);
+  }
+
+  @return $_stops;
+}
+
+
+// Get Asymmetrical Outside
+// ------------------------
+// - $grid: <settings>
+@function get-asym-outside(
+  $grid
+) {
+  $grid     : parse-grid($grid);
+  $_columns : susy-get(columns, $grid);
+
+  $_color   : debug-get(color);
+  $_light   : lighten($_color, 15%);
+  $_trans   : transparent;
+  $_stops   : ();
+
+  @for $location from 1 through susy-count($_columns) {
+    $this-stop: ();
+
+    @if $location == 1 {
+      $this-stop: append($this-stop, $_color, comma);
+    } @else {
+      $start: parse-span(1 at $location $grid);
+      $start: get-isolation($start);
+      $this-stop: append($this-stop, $_color $start, comma);
+    }
+
+    @if $location == susy-count($_columns) {
+      $this-stop: append($this-stop, $_light, comma);
+    } @else {
+      $gutter: get-span-width(first $location $grid);
+
+      $_end: parse-span(1 at ($location + 1) $grid);
+      $_end: get-isolation($_end);
+
+      $gutter: $_light $gutter, $_trans $gutter, $_trans $_end;
+      $this-stop: join($this-stop, $gutter, comma);
+    }
+
+    $_stops: join($_stops, $this-stop, comma);
+  }
+
+  @return $_stops;
+}
+
+
+// Get Asymmetrical Background
+// ---------------------------
+// - $grid: <settings>
+@function get-background-asym(
+  $grid
+) {
+  $_stops: ();
+
+  @if is-inside($grid) {
+    $_stops: get-asym-inside($grid);
+  } @else if is-split($grid) {
+    $_stops: get-asym-split($grid);
+  } @else {
+    $_stops: get-asym-outside($grid);
+  }
+
+  @return (image: (columns: $_stops));
+}
+
+
+// Get Background
+// --------------
+// - $grid: <settings>
+@function get-background(
+  $grid
+) {
+  $grid     : parse-grid($grid);
+  $_show    : susy-get(debug image, $grid);
+  $_return  : ();
+
+  @if $_show and $_show != hide {
+    $_line-height: variable-exists(base-line-height);
+    $_line-height: if($_line-height, $base-line-height, false);
+    $_columns: susy-get(columns, $grid);
+
+    @if $_show != show-baseline {
+      $_sym: is-symmetrical($_columns);
+      $_return: if($_sym, get-background-sym($grid), get-background-asym($grid));
+      $_return: map-merge($_return, (clip: content-box));
+    } @else if not($_line-height) {
+      @warn 'Please provide $base-line-height in order to see baseline grids.';
+    }
+
+    @if $_line-height {
+      @if $_show != show-columns {
+        $_color: variable-exists(grid-background-baseline-color);
+        $_color: if($_color, $grid-background-baseline-color, #000);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.